runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessDetailsDialog/ProcessDetailsTabs/CredentialsTab.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+
+import { Box, Chip, Typography } from '@mui/material';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import { useDispatch, useSelector } from '#src-app/store';
+import { processActions, processSelector } from '#src-app/store/slices/Process';
+
+import {
+    DetailsInfoTabProps,
+    ProcessDetailsTab,
+} from '../ProcessDetailsDialog.types';
+
+export const CredentialsTab = ({ value, process }: DetailsInfoTabProps) => {
```

Comment:
this component has the same structure like the AccessTab component (same styling, but data is different - do you think it is possible to merge it into one component?